src/main/java/me/ironexception/universalfriends/association/Association.java

```diff
@@ -53,7 +53,7 @@ public static Association byExactValue(final double value) {
     }
 
     /**
-     * Fetch the {@link Association} convert value to association.
+     * Fetch a {@link Association} by what the value means.
      * No matter how friendly someone is as long as he is friendly he gets the ally association. The same goes for enemies.
      * So values below 0 are enemies and above are allies.
```

Comment:
This still doesn't express what this method does in a precise manner.

Please consider the example I provided with method in my previous review.